pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/DistinctTable.java

```diff
@@ -39,85 +44,54 @@
  * (2) The same object is serialized by the server inside the data table
  * for sending the results to broker. Broker deserializes it.
  */
-public class DistinctTable {
+public class DistinctTable extends BaseTable {
```

Comment:
Yes I want to do that but since that would result in boilerplate changes (this file will be shown as deleted and all the callers will have one line changes s/DistinctTable/SetTable) so I decided not to pollute this PR and do that in follow-up.